src/System.Net.WebSockets/src/System/Net/WebSockets/WebSocketException.cs

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.Net.WebSockets
+{
+    public sealed class WebSocketException : Win32Exception
+    {
+        private WebSocketError _webSocketErrorCode;
+
+        [SuppressMessage("Microsoft.Security", "CA2122:DoNotIndirectlyExposeMethodsWithLinkDemands",
+            Justification = "This ctor is harmless, because it does not pass arbitrary data into the native code.")]
```

Comment:
We asked this question during the pre-review.  @mconnew said that it was applicable still.

But it would be good to get more confirmation.

@ericstj - Can you confirm?